ui/src/components/PipelineGraph/RawOrTopologyControl.tsx

```diff
@@ -0,0 +1,57 @@
+import { ToggleButton, ToggleButtonGroup } from "@mui/material";
+
+interface RawOrTopologyControlProps {
+  rawOrTopology: "raw" | "topology";
+  setTopologyOrRaw: React.Dispatch<React.SetStateAction<"topology" | "raw">>;
+}
+export const RawOrTopologyControl: React.FC<RawOrTopologyControlProps> = ({
+  rawOrTopology,
+  setTopologyOrRaw,
+}) => {
+  
+  const toggleRawOrTopology = (
+    _: React.MouseEvent<HTMLElement>,
+    newValue: "topology" | "raw",
+    ) => {
+      setTopologyOrRaw(newValue);
+  };
+  return (
+    <ToggleButtonGroup
+      sx={{
+        display: 'flex',
+        justifyContent: 'center',                  
```

Comment:
```suggestion
        justifyContent: 'center',
```
There are a few lines in here with a bunch of trailing spaces